src/components/filter-modal/FilterModal.jsx

```diff
@@ -0,0 +1,107 @@
+import { useNotesContext } from "context";
+import "./FilterModal.css";
+
+const FilterModal = ({ setIsFilter }) => {
+  const {
+    notesState: { label, priority, sortByLatest },
+    notesDispatch,
+  } = useNotesContext();
+
+  const listOfLabels = ["office", "food", "home"];
+  const listOfPriorities = ["low", "medium", "high"];
+
+  return (
+    <div className="filter">
+      <div className="flex space-between">
+        <h4>Tags:</h4>
+        <button
+          className="btn btn-clear"
+          onClick={() => {
+            notesDispatch({ type: "RESET_FILTERS" });
+            setIsFilter(false);
+          }}
+        >
+          clear all
+        </button>
+      </div>
+      <div className="flex gap-1rem flex-start">
+        {listOfLabels.map((singleLabel) => (
+          <label
+            key={singleLabel}
+            htmlFor={`${singleLabel}-filter`}
+            className="note-modal-label"
+          >
+            <input
+              type="checkbox"
+              id={`${singleLabel}-filter`}
+              checked={label[singleLabel] === true}
+              onChange={(e) =>
+                notesDispatch({
+                  type: "SET_LABEL",
+                  payload: { [singleLabel]: e.target.checked },
+                })
+              }
+            />
+            {singleLabel}
+          </label>
+        ))}
+      </div>
+      <h4>Priorities:</h4>
+      <div className="flex gap-1rem flex-start">
+        {listOfPriorities.map((singlePriority) => (
```

Comment:
Same suggestion here, try to use ids for the key attribute to avoid man made errors.